package.json

```diff
@@ -4,7 +4,7 @@
   "keywords": [
     "assert"
   ],
-  "version": "1.3.0",
+  "version": "1.4.0",
```

Comment:
PRs shouldn't have version bumps, do those separately. This allows you to merge several PRs without deciding that the version change is yet.